src/main/java/gov/usgs/wma/waterdata/ProcessTimeSeriesDescription.java

```diff
@@ -31,7 +31,7 @@ protected ResultObject processRequest(RequestObject request) {
 		ResultObject result = new ResultObject();
 
 		if (null != jsonDataId) {
-			List<TimeSeries> timeSeriesList = tsdDao.upsertTimeSeriesDescription(jsonDataId);
+			List<TimeSeries> timeSeriesList = tsdDao.upsertTimeSeriesDescription(jsonDataId, request.getPartitionNumber());
```

Comment:
Not sure why the null check on jsonDataID and not partitionNumber. Maybe just remove the check?
Same thought as the ts-corrected on sending individual values vs the object in method calls.

Reply:
Good idea - I'll do something like what I described in the ts-corrected PR comment and remove the null check - I think this just sends us into the error handler unnecessarily at this point?